assets/js/modules/sign-in-with-google/components/dashboard/SetupSuccessSubtleNotification.stories.js

```diff
@@ -51,11 +51,17 @@ export default {
 	component: SetupSuccessSubtleNotification,
 	decorators: [
 		withQuery,
-		( Story ) => {
+		( Story, { args } ) => {
+			const setupRegistry = ( registry ) => {
+				if ( args?.setupRegistry ) {
+					args.setupRegistry( registry );
+				}
+			};
+
 			return (
-				<WithTestRegistry>
```

Comment:
Any decorator like this with no setup function can be removed as no longer necessary.